### ISSUE TITLE
Marker clusterer bug when dragging the map

### ISSUE THREAD

**Issue by sanford-opspl** [googlemaps/v3-utility-library#618]
Okay, I'm developing an Ionic 3 app and implemented the Google Maps along with Marker Clustering too. I noticed that when I drag the map, there is an issue. If I start dragging the map while my finger is on one of the marker clusters, the map cannot be dragged. But if I do the same outside the area, it works fine.

Am I missing something or this bug actually exists? Kindly help.

**Comment by jpoehnelt** [comment]
might be duplicate of #506

**Comment by dcts** [comment]
yes i noiced the same thing. It happens only on mobile, desktop works fine. I tried to fix it but the only option I could find is to bypass the pointer events on the cluster like this:
```css
.cluster {
  pointer-events: none !important;
}
```

This is not ideal at all, since then of course you also disable the click events on the cluster, hence loosing a great functionality. Or is there a way to prevent all pointerevents but allow only the click event?
